bin/g5k-checks

```diff
@@ -38,7 +38,7 @@ optparse = OptionParser.new(USAGE) do |opts|
   end
 
   $hashconf["mode"] = "oar_checks"
-  opts.on( '-m', '--mode MODE', "Specify the functional mode (oar_checks, api, jenkins) [default: 'oar_checks']") do |mode|
+  opts.on( '-m', '--mode MODE', "Specify the functional mode (oar_checks, api, jenkins, no_api) [default: 'oar_checks']") do |mode|
```

Comment:
I don't think we need this.

Reply:
Removed, good catch! This was left overs from when I was trying to make a new mode.